index.html

```diff
@@ -72,9 +77,37 @@ <h1>MTA Subway Time</h1>
 
         <script>
             const hamburgerButton = document.querySelector(".js-hamburger");
+            const menu = document.querySelector(".menu")
+            const home = document.querySelector("#home")
```

Comment:
What would happen if we added a new link to the `<nav>`? 

```html
<nav>
  <ul>
    <li>home</li>
    <li>subway map</li>
    <li>awesome feature</li>
  </ul>
</nav>
```

We would have to update our JavaScript code to select the `awesome feature` element, rather than the `subway map` element. 

One way to future proof this is to get the first and last element in the `<nav>`.

```diff
- const home = document.querySelector("#home");
- const subway_map = document.querySelector("#subway_map");
+ const menuLinks = menu.querySelectorAll("a");
+ const firstEle = menuLinks[0];
+ const lastEle = menuLinks[menuLinks.length - 1];

Reply:
Good advice! Earlier I saw this method first and customized it to our two element version to be more specific.